pkg/workloads/console/integration/integration_test.go

```diff
@@ -66,37 +88,211 @@ var _ = Describe("Console", func() {
 
 	Describe("Creating resources", func() {
 		It("Sets console.spec.user from rbac", func() {
+			consoleTemplate := &workloadsv1alpha1.ConsoleTemplate{
```

Comment:
Shall we move this standard console template into a helper function? Something like this: https://github.com/gocardless/theatre/blob/master/pkg/rbac/directoryrolebinding/integration/controller_test.go#L27-L41

It's reused across tests, so I reckon this probably makes things a bit clearer.

Reply:
We could even create it in the BeforeEach so we can assume the template is present in each test. Given we're actually testing creation of consoles, rather than the templates, that may make sense. It's what we do for the [admin role](https://github.com/gocardless/theatre/blob/master/pkg/rbac/directoryrolebinding/integration/controller_test.go#L91-L94) in the drb specs.